ceph-salt-formula/salt/ceph-salt/files/registries.conf.j2

```diff
@@ -0,0 +1,18 @@
+{% set registries = pillar['ceph-salt'].get('container', {}).get('registries', []) -%}
+# System Registry Configuration File (VERSION 2)
```

Comment:
- Do we usually have a header to mention that the file is managed by salt?
- nit: can we have as first line `# For more information on this configuration file, see containers-registries.conf(5).` ?

Reply:
Fixed.  Add a header on template based files